src/v/archival/ntp_archiver_service.cc

```diff
@@ -102,7 +102,10 @@ ntp_archiver::download_manifest(retry_chain_node& parent) {
     auto path = _manifest.get_manifest_path();
     auto key = cloud_storage::remote_manifest_path(
       std::filesystem::path(std::move(path)));
-    co_return co_await _remote.download_manifest(_bucket, key, _manifest, fib);
+    bool supress_not_available_message
+      = (_partition->high_watermark() == model::offset(0));
+    co_return co_await _remote.download_manifest(
```

Comment:
Maybe we can just skip the attempt to download the manifests in this case? @ztlpn what do you think?

Reply:
As discussed offline, decided not to change the download logic now since there maybe be implications for recovery.